Add mention of the CC (#5)
tari-project/rfcs: src/RFC-0312_DANHighLevelSpecification.md

The asset issuer will also draft and publish the [contract constitution]. The constitution defines _how_ a contract is
run, and defines the conditions under which the terms of the constitution can be changed.

### The Constitution Committee
[constitution committee]: #constitution_committee
[CC]: #cc

> In the [asset issuer] will in the creation of the [contract constitution] nominate a key or set of keys to "own" the asset

Reviewer: Something is wrong with the grammar here

Author: thanks fixed

and control all things related to _how_ the contract is run. They are known as the [constitution committee] (CC)

> The CC has the power to change everything inside of the [contract constitution]. In most cases this will simply by the [asset issuer]

Reviewer 2: ```suggestion
The CC has the power to change anything inside of the [contract constitution]. In many cases, the CC will simply be the [asset issuer].
```

> but this can be any number of different cases from a DAO, nominated list of keys to a single specific key.

Reviewer 2: ```suggestion
However, allowing the CC to differ from the asset issuer enables a number of other use-cases 
such as a DAO, a nominated list of keys, etc.
```

### Changes to the constitution
[contract constitution change]: #changes-to-the-constitution

> Any changes to the [contract constitution] can happen at any time. This also refers to any changes to the VNC. Only the [CC] may 

Reviewer 2: `[constitution amendment]` needs to be linked to the description of the process.
```suggestion
Changes to the [contract constitution] can happen at any time through the [constitution amendment] process. This also refers to any changes to the VNC. Only the [CC] may 
```

make changes to the[contract constitution].

* The rules over how members are added or removed are defined in the [contract constitution].
* At the minimum, there's a proposal step, a validation step, an acceptance step, and an activation step. Therefore
  changes take place over at least a 4-checkpoint time span.
* If a VN leaves a committee their [side-chain deposit] MAY be refunded to them.
* If a new VN joins the committee they must provide the [side-chain deposit] at their activation step.

> * In the proposal step, any authorised [CC] may make a change proposal, as defined in the [contract constitution]

Reviewer 2: ```suggestion
* In the proposal step, any authorised [CC] may make a change proposal, within the limits defined by the change rules in the [contract constitution]
```

> * Before activation all VNC members needs to submit an acceptance transaction again, or resign by default if none is submitted. This 
>   gives them the opportunity to resign if they do not agree with the new rules as specified in the constitution.

Reviewer 2: ```suggestion
* Before activation, VNC members MAY submit an acceptance transaction that registers their willingness to validate the contract. If no acceptance is submitted within the `acceptance_period` the validator is assumed to be uninterested in running the contract and will not form part of the finalized contract committee. 
```
